### PR TITLE
Fix failing buteo-mtp unit tests

### PR DESCRIPTION
**Pull request by spiiroin**

### CODE REVIEW

File: mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp [hunk 1]
```diff
@@ -38,149 +38,78 @@
...
 
 using namespace meegomtp1dot0;
 
 int totalCount = 17;
 
-static const int THUMBNAIL_WIDTH = 100;
-static const int THUMBNAIL_HEIGHT = 100;
+/* These need to match THUMB_SIZE in thumbnailer.cpp */
+static const int THUMBNAIL_WIDTH = 128;
+static const int THUMBNAIL_HEIGHT = 128;;
```
Comment (pvuorela) [inline mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp]: Extra semicolon.

File: mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp [hunk 2]
```diff
@@ -213,8 +142,9 @@ void FSStoragePlugin_test::testDeleteAll()
     QCOMPARE( m_storage->m_objectHandlesMap.size(), m_storage->m_pathNamesMap.size() );
 
     delete m_storage;
+    m_storage = nullptr;
     system("rm -rf ~/.local/mtp");
-    system("rm -rf /tmp/mtptests");
+    QVERIFY( QDir(STORAGE1).removeRecursively() );
     initTestCase();
```
Comment (pvuorela) [inline mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp]: Old, but calling initTestCase() from a test case (and then another test case) smells a bit bad.

**Review by pvuorela — APPROVED** [review]
Tests pass and the changes look sane.

**Comment by pvuorela** [comment]
One more thing: while at it, could you rename the test set name in tests.xml to something better than "libmeegomtp-tests"

**Comment by pvuorela** [comment]
Would be nice if such meegomtp naming wasn't used anywhere here but adjusting the test set already makes the test run results nicer.

**Comment by pvuorela** [comment]
Still passes and looks even better -> still approved.

Please just squash some commits before merging.

File: mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp (hunk 2 above)
Comment (spiiroin) [reply mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp]: fixed

File: mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp (hunk 1 above)
Comment (spiiroin) [reply mts/platform/storage/fsstorageplugin/unittests/fsstorageplugin_test.cpp]: fixed

**Comment by spiiroin** [comment]
Old review commits squashed.
Found problem with responder test: leftover files can cause following tests to fail -> fix for that added.

**Comment by pvuorela** [comment]
Still good.

**Comment by spiiroin** [comment]
Review commits squashed.

**Pull request merged**